### PR TITLE
Ankit | Test | Provide "Other" option with text field support on create company page

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cwdjz5w

### CODE REVIEW

File: apps/web-giddh/src/app/add-company/add-company.component.ts [hunk 1]
```diff
@@ -609,6 +610,7 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
             businessType: [''],
             businessNature: [''],
             gstin: [''],
+            otherBusinessNature:[''],
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/add-company/add-company.component.ts]:
```suggestion
            otherBusinessNature: [''],
```

File: apps/web-giddh/src/app/add-company/add-company.component.ts [hunk 2]
```diff
@@ -1159,6 +1161,7 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
         this.socketCompanyRequest.utm_campaign = this.generalService.getUtmParameter('utm_campaign');
         this.socketCompanyRequest.utm_term = this.generalService.getUtmParameter('utm_term');
         this.socketCompanyRequest.utm_content = this.generalService.getUtmParameter('utm_content');
+        this.socketCompanyRequest.BusinessNature = this.secondStepForm.controls['otherBusinessNature'].value;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/add-company/add-company.component.ts]: if (this.secondStepForm.value.businessNature === "Other") {

Comment (dvCodeWorld) [reply apps/web-giddh/src/app/add-company/add-company.component.ts]: use this

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/add-company/add-company.component.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/add-company/add-company.component.ts]: Done

File: apps/web-giddh/src/app/add-company/add-company.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/add-company/add-company.component.ts]: Done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**